elements/jsonform/stories/public/collectionSchema.json

```diff
@@ -632,14 +632,7 @@
               "properties": {
                 "bbox": {
                   "type": "array",
-                  "items": {
-                    "type": "array",
-                    "format": "bounding-box",
-                    "items": { "type": "number" },
-                    "minItems": 4,
-                    "maxItems": 4
-                  },
-                  "minItems": 1
+                  "format": "bounding-boxes"
```

Comment:
One small adjustment after some research: the `bounding-boxes` format would move to this "level", as you correctly intended, and is therefore not needed any longer in line 630 (one level "higher"). The rest of the schema should probably stay the same as before, following the (simpliefied) [STAC JSON schema](https://github.com/radiantearth/stac-spec/blob/ec002bb93dbfa47976822def8f11b2861775b662/collection-spec/json-schema/collection.json#L69). So, in total the `spatial` property would look like this:

```json
            "spatial": {
              "title": "Spatial Extents",
              "type": "object",
              "required": ["bbox"],
              "properties": {
                "bbox": {
                  "type": "array",
                  "format": "bounding-boxes",
                  "items": {
                    "type": "array",
                    "format": "bounding-box",
                    "items": { "type": "number" },
                    "minItems": 4,
                    "maxItems": 4
                  },
                  "minItems": 1
                }
              }
            },
```

This renders nicely the map with multi select and correctly sets the values!
![image](https://github.com/user-attachments/assets/a14783d8-4e78-4acc-8956-8415b7620421)